src/components/tables/table-children/TableChildBody.tsx

```diff
@@ -26,22 +25,25 @@ export interface ITableChildBodyProps extends ITableBodyInheritedFromTableProps
     rowData: IData;
     isLoading: boolean;
     onRowClick?: (actions: IActionOptions[]) => void;
+    disabled?: boolean;
     isMultiSelect: boolean;
     withoutHoverOnRow?: boolean;
     handleOnRowClick?: (actions: IActionOptions[], rowData: IData) => void;
 }
 
 export const TableChildBody = (props: ITableChildBodyProps): JSX.Element => {
     const headingAndCollapsibleId = `${getTableChildComponentId(props.tableId, TableChildComponent.TABLE_HEADING_ROW)}${props.rowData.id}`;
-    const tableHeadingRowContent = props.headingAttributes.map((headingAttribute: ITableHeadingAttribute, xPosition: number) => {
+    const tableHeadingRowContent = _.map(props.headingAttributes, (headingAttribute: ITableHeadingAttribute, xPosition: number) => {

```

Comment:
we leave it to the developer to use underscore or not, but I would recommend that when one is used, do not revert to the other one by personal preference because it creates noise in the PR unnecessarily